compiler/damlc/daml-stdlib-src/DA/Internal/Interface.daml

```diff
@@ -22,13 +21,18 @@ import DA.Internal.Template.Functions
 import DA.Internal.LF
 import DA.Internal.Any
 
+-- | (1.dev only) Exposes the `interfaceTypeRep` function. Available only for interfaces.
```

Comment:
just thinking, it would be nice if we had something like
```
#if MIN_VERSION_DAML_LF(1,"DEV")
```

Reply:
We can use `#ifdef DAML_INTERFACES` for similar effect. The reason not to use that here is because desugaring is agnostic of the LF version (for now ... hopefully forever), so we kind of want these typeclasses to exist in all versions, and then hopefully give better errors in the compiler when people try to implement interfaces outside of 1.dev.